index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,300&display=swap"
+      rel="stylesheet"
+    />
+    <link href="css/index.css" rel="stylesheet" />
+    <title>AGE-21</title>
+    <script
+      src="https://kit.fontawesome.com/b99221d6be.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body data-bs-spy="scroll" data-bs-target="#navbar-spy" data-bs-offset="48">
+    <div class="container">
+      <header class="appbar bg">
+        <nav id="navbar-spy" class="navbar">
+          <a class="navbar-brand logo" href="./">AGE<span>Conf</span></a>
+          <button id="open-menu" class="navbar-toggler" type="button">
+            <i class="fas fa-bars"></i>
+          </button>
+          <ul class="nav navbar-nav color-dark-text">
+            <li class="nav-item">
+              <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#about">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="join-nav-button color-primary bd-primary" href="#join"
+                >Join</a
+              >
+            </li>
+          </ul>
+        </nav>
+      </header>
```

Comment:
**NAVBAR**
**_On Mobile Size:_**

- [ ] The background of the navbar should be transparent or the same color/pattern as the top section (headline) background.

**_On Desktop Size:_**
- [ ] According to the design, It is divided into a top dark bar and a bottom white bar.
- [ ] The top short bar section is completely missing: it consists of the social media links, logout, and language links aligned to the right.